packages/cassowary/lib/cassowary.dart

```diff
@@ -3,6 +3,8 @@
 // found in the LICENSE file.
 
 /// An implementation of the Cassowary constraint solving algorithm in Dart.
+/// 
+/// To use, import `package:cassowary/cassowary.dart`.
```

Comment:
The other's just need a package: ref in the code; this and newton will in addition need a pubspec entry (the `../flutter/flutter/packages/flutter` entry already exists for users).